metrics/config_observability.go

```diff
@@ -100,6 +103,7 @@ func NewObservabilityConfigFromConfigMap(configMap *corev1.ConfigMap) (*Observab
 		cm.AsBool(EnableReqLogKey, &oc.EnableRequestLog),
 		cm.AsBool(EnableProbeReqLogKey, &oc.EnableProbeRequestLog),
 		cm.AsString("metrics.request-metrics-backend-destination", &oc.RequestMetricsBackend),
+		cm.AsInt("metrics.request-metrics-reporting-period-seconds", &oc.RequestMetricsReportingPeriodSeconds),
```

Comment:
Not using internal key `metrics.reporting-period-seconds` to target specifically queue proxy.

Reply:
Wonder if it's worth adding this context to the actual field / as a comment here -- if it's important enough for a reviewer here to know it might be something a future reader of the code might want to know too

Reply:
Yes would be helpful.

Reply:
Gave a second though to this. We dont need that this is meant for queue proxy. We can do this at the side where we use the field. For example ` metrics.request-metrics-backend-destination` use in Serving is documented here:

https://github.com/knative/serving/blob/b70e8979d8b61e5d0dfa6eb3b9387cc789adb437/config/core/configmaps/observability.yaml#L99-102